how_to_eurec4a/specmacs.md

```diff
@@ -198,119 +254,154 @@ def e2(a, b):
     f = 1 - (b/a)
     e_squared = 2*f - f**2
     return e_squared
+
+def ellipsoidal_to_ecef_ds(ds: xr.Dataset) -> xr.DataArray:
+    '''
+    This function applies the conversion from above directly to an xarray dataset.
+    '''
+    return xr.apply_ufunc(ellipsoidal_to_ecef, np.deg2rad(ds.lat), np.deg2rad(ds.lon), ds.alt,
+                          output_core_dims=[("xyz_ecef",)])
 ```
 
-With these functions it is easy to transform the lat/lon/height position of the HALO into the ECEF-coordinate system:
+With these functions it is easy to transform the lat/lon/height position of the HALO into the ECEF-coordinate system.
```

Comment:
I changed it, because `alt` is the name of the variable in the dataset. But height is used everywhere in the computation... Probably we need some words about why this is the same?

Reply:
I don't think that we need some words on why height in the computation and alt are the same because `height` in the computation is just a parameter in a function which does not necessarily have to have the same notation as the variable. 

Maybe we could stick to alt/altitude when talking about the altitude of the HALO and to height when talking about the cloud height?

Reply:
So do you also think that we should change `height` to `alt` within the navipedia functions?